airflow/utils/dag_processing.py

```diff
@@ -277,7 +277,7 @@ def get_dag(self, dag_id):
 
 
 def list_py_file_paths(directory, safe_mode=True,
-                       include_examples=conf.getboolean('core', 'LOAD_EXAMPLES')):
+                       include_examples=None):
```

Comment:
This change makes no difference to me.

Reply:
Hi @feng-tao , understand this PR is merged and it does bring significant improvement (big thanks to @astahlman !).

But may you check my comment? It's a very minor point, but not really necessary to add a separate check here (Kindly let me know if I missed anything). Thanks.

Reply:
This was needed because of [the way that Python handles default arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments): the default value is evaluated when the function is defined, not when it is called. Therefore, our mocked implementation of `getboolean` wouldn't get called here, and `include_examples` will be always be `True`.

This way, we call `getboolean` every time the function is evaluated, so our mock implementation does get used and `getboolean('core', 'LOAD_EXAMPLES')` evaluates to `False`.

Reply:
Get it. Thanks for the clarification @astahlman